client/components/public/imports/HomeIntro.jsx

```diff
@@ -34,31 +34,36 @@ export default class HomeIntro extends Component {
+                  {/* <p>The Great Puzzle Hunt is like a scavenger hunt adventure with puzzles. Teams of up to 6 people travel on foot to various locations solving a total of four hour-long puzzles gathering clues along the way to solve one final meta puzzle.</p>
+                  <p>These are no ordinary puzzles though! It will take a diverse set of skills and talents to solve them! Our mission is to celebrate everyone's talents and demonstrate knowledge comes in many forms. Everything is timed using your phone, QR codes, and this online system! So yes you can win :)</p> */}
+                  <p>Teams of up to 6 people solve a total of four hour-long puzzles gathering clues along the way to solve one final meta puzzle.</p>
+                  <p>These are no ordinary puzzles though! It will take a diverse set of skills and talents to solve them! Our mission is to celebrate everyone's talents and demonstrate knowledge comes in many forms. Everything is timed using your phone, QR codes, and our online game platform! So yes, you can win! :)</p>
                 </Segment>
               </Container>
             </Grid.Column>
           </Grid.Row>
 
           <Grid.Row columns={3}>
             <Grid.Column>
-              <Segment  inverted color='blue' >
+              <Segment padded inverted color='blue' >
                 <Header as='h1' size="medium">Who?</Header>
-                Everyone! However, each person under age 14 on a team must be accompanied by a parent/legal guardian at all times - parent/legal guardian must be registered on team with child under age 14. *Note: The puzzles are created for ages 14 and older.
+                {/* Everyone! However, each person under age 14 on a team must be accompanied by a parent/legal guardian at all times - parent/legal guardian must be registered on team with child under age 14. *Note: The puzzles are created for ages 14 and older. */}
+                Everyone! However, each participant under age 14 must have permission from a parent/legal guardian. The puzzles are created for ages 14 and older.
               </Segment>
               <Segment padded inverted color='blue' >
                 <Header as='h1' size="medium">Where?</Header>
-                Red Square<br/>
+                {/* Red Square<br/>
                 Western Washington University<br/>
                 516 High Street<br/>
-                Bellingham, WA 98225<br/><br/>
+                Bellingham, WA 98225<br/><br/> */}
+                <p>The 2021 Puzzle Hunt is virtual, so anywhere that has WiFi access, a smartphone or computer, a black and white printer, and materials listed under the FAQs under <a href="/faq" style={{color: "#bad80a"}}>What should I bring?</a></p>
```

Comment:
nit: `WiFi` -> `internet`. I wouldn't normally care, but this event is targeted towards nerds, and some of them WILL care about the difference! ;)

Reply:
Fixed!